assets/aurora.tokenlist.json

```diff
@@ -113,18 +113,6 @@
         "allbridge"
       ]
     },
-    {
-      "chainId": 1313161554,
-      "address": "0x0fAD0ED848A7A16526E8a7574e418B015Dbf41B5",
-      "symbol": "PAD",
-      "name": "SmartPad Token - Allbridge",
-      "decimals": 18,
-      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/abr_smartpad.svg",
-      "tags": [
-        "ethereum",
-        "allbridge"
-      ]
-    },
```

Comment:
Please don't manually edit this file. This will be automatically regenerated along with bumping up token list version